### ISSUE TITLE
BfME IniParseException

### ISSUE THREAD

**Issue by MinecraftPlaye** [OpenSAGE/OpenSAGE#149]
I don't know if these Games are already implemented, but whenever I'm opening the Launcher I'll get the following error:
```
OpenSAGE was unable to find any installations of Command & Conquer (tm): Generals.

You can manually specify the installation path by setting the following environment variable:
        CNC_GENERALS_PATH=<installation path>

OpenSAGE doesn't yet detect every released version of every game. Please report undetected versions to our GitHub page:
        https://github.com/OpenSAGE/OpenSAGE/issues
```

Whenever I'm opening the Viewer I'm getting this error:

```
Unhandled Exception: OpenSage.Data.Ini.Parser.IniParseException: (G:\Games\BfMEGames\TBfMe2\data\ini\playertemplate.ini,66,2): Unexpected field 'Evil' in block 'PlayerTemplate'.
   at OpenSage.Data.Ini.Parser.IniParser.ParseBlockContent[T](T result, IIniFieldParserProvider`1 fieldParserProvider)
   at OpenSage.Data.Ini.Parser.IniParser.ParseNamedBlock[T](Action`2 setNameCallback, IIniFieldParserProvider`1 fieldParserProvider)
   at OpenSage.Data.Ini.Parser.IniParser.<>c.<.cctor>b__70_69(IniParser parser, IniDataContext context)
   at OpenSage.Data.Ini.Parser.IniParser.ParseFile()
   at OpenSage.Data.Ini.IniDataContext.LoadIniFile(FileSystemEntry entry)
   at OpenSage.Content.ContentManager..ctor(Game game, FileSystem fileSystem, GraphicsDevice graphicsDevice, SageGame sageGame, WndCallbackResolver wndCallbackResolver)
   at OpenSage.Game..ctor(IGameDefinition definition, FileSystem fileSystem, GamePanel panel)
   at OpenSage.Viewer.UI.MainForm.ChangeInstallation(GameInstallation installation)
   at OpenSage.Viewer.UI.MainForm..ctor(GameWindow gameWindow, ImGuiRenderer imGuiRenderer)
   at OpenSage.Viewer.Program.Run(Options opts)
   at CommandLine.ParserResultExtensions.WithParsed[T](ParserResult`1 result, Action`1 action)
   at OpenSage.Viewer.Program.Main(String[] args)
```

Installed Games (Anthology DVD version):
- BfME
- BfME II
- BfME II RotWk (with Edain Mod)

I'm running v0.3.0 of OpenSAGE on Windows 10 x64.

**Comment by feliwir** [comment]
Yes, those games are implemented for the viewer. This is a known bug and was fixed in #145 by me. We will hopefully do a new release with these changes soon.

**Comment by MinecraftPlaye** [comment]
> 
> 
> Yes, those games are implemented for the viewer. This is a known bug and was fixed in #145 . We will hopefully do a new release with these changes soon. Please note that those games are not fully working yet,

Ok, guess I'll have to wait for the next release. :)

**Comment by tgjones** [comment]
Since this is already fixed, I’m going to it. If we were more organised we’d assign closed issues to milestones so it’s clear when a bug fix or feature gets released.
